src/battle/common/actor/goomba.inc.c

```diff
@@ -119,7 +119,7 @@ EvtSource N(idle_80219938) = {
     EVT_LOOP(LW(0))
         EVT_LABEL(0)
         EVT_CALL(GetStatusFlags, ACTOR_SELF, LW(1))
-        EVT_IF_FLAG(LW(1), 0x35D000)
+        EVT_IF_FLAG(LW(1), (STATUS_FLAG_SLEEP | STATUS_FLAG_FROZEN | STATUS_FLAG_FEAR | STATUS_FLAG_PARALYZE | STATUS_FLAG_DIZZY | STATUS_FLAG_STONE | STATUS_FLAG_STOP))
```

Comment:
I'm pretty sure these parens aren't required anymore -- the double parens originally in flags were to tell the SCRIPT compiler things